.changeset/gold-days-end.md

```diff
@@ -3,3 +3,5 @@
 ---
 
 Added `activeDescription` optional property to TreeItemAction interface, which when provided a value will display a dot above the button indicating an active state. The provided value is used to set accesible description and should explain why the action item is active.
+
+Renamed `createFilterAction` to `useFilterAction`.
```

Comment:
This will create a second paragraph to the `activeDescription` change. Instead, we want the rename to appear as a separate change in the changelog - you'll need to create a separate change file for that.

Reply:
Did not know you can have multiple change files for the same package, changed to a separate one.